.gitignore

```diff
@@ -51,4 +51,5 @@ phpcs.xml
 /tools/docker/bin/jt/
 
 # This file indicates we're in draft mode, which reduces checks
-.jetpack-draft
\ No newline at end of file
+.jetpack-draft
+.git-hook-data
```

Comment:
Since that's not related to Draft mode, do you think you could place this above that last comment?

Reply:
Thanks for spotting that, I've moved `.git-hook-data` above.